NEWS.md

```diff
@@ -6,113 +6,98 @@ New language features
     constructors ([#30577]).
+  * Support for Unicode 12.0.0 ([#31561]).
+  * Added `⋆` (`\star`) as unary operator ([#31604]).
 
-  * Objects created by calling `skipmissing` on an array can now be indexed using indices
-    from the parent at non-missing positions. This allows functions such as
-    `findall`, `findfirst`, `argmin`/`argmax` and `findmin`/`findmax` to work with these
-    objects, returning the index of matching non-missing elements in the parent ([#31008]).
-
-  * `inv(::Missing)` has now been added and returns `missing` ([#31451]).
-
-  * `nextfloat(::BigFloat, n::Integer)` and `prevfloat(::BigFloat, n::Integer)` methods
-    have been added ([#31310]).
+Language changes
+----------------
 
-  * Support for Unicode 12.0.0 ([#31561]).
+  * Empty entries in `JULIA_DEPOT_PATH` are now expanded to default depot entries ([#31009]).
 
 Multi-threading changes
 -----------------------
 
-* The `Condition` type now has a thread-safe replacement, accessed as `Threads.Condition`.
+  * The `Condition` type now has a thread-safe replacement, accessed as `Threads.Condition`.
     With that addition, task scheduling primitives such as `ReentrantLock` are now thread-safe ([#30061]).
-
   * It is possible to schedule and switch Tasks during `@threads` loops, and perform limited I/O ([#31438]).
 
-Language changes
-----------------
-* Empty entries in `JULIA_DEPOT_PATH` are now expanded to default depot entries ([#31009]).
-* `Enum` now behaves like a scalar when used in broadcasting ([#30670]).
-* If a `pipeline` is specified with `append=true` set, but no redirection, an `ArgumentError`
-is thrown, rather than a `ErrorException` ([#27900]).
-* Functions that invoke commands (e.g. `run(::Cmd)`) now throw a `ProcessFailedException`
-rather than an `ErrorException`, if those commands exit with non-zero exit code.
-([#27900]).
-* Added `⋆` (`\star`) as unary operator ([#31604]).
-
-Command-line option changes
----------------------------
+Build system changes
+--------------------
 
+  * The build system now prefers downloading prebuilt binary tarballs for most dependencies on
+    supported systems, disable by setting `USE_BINARYBUILDER=0` at `make` time ([#31441]).
 
 New library functions
 ---------------------
 
-* `getipaddrs()` function returns all the IP addresses of the local machine, with IPv4 addresses sorting before IPv6 addresses ([#30349, #30604])
-* `getipaddr(addr_type)` and `getipaddrs(addr_type)` functions returns an IP address(es) of the desired type of the local machine ([#30604])
-* Added `Base.hasproperty` and `Base.hasfield` ([#28850]).
-* One argument `!=(x)`, `>(x)`, `>=(x)`, `<(x)`, `<=(x)` has been added for currying,
-  similar to the existing `==(x)` and `isequal(x)` methods ([#30915]).
+  * `getipaddrs()` function returns all the IP addresses of the local machine, with IPv4 addresses sorting before IPv6 addresses ([#30349, #30604]).
```

Comment:
I'd really like for us to move toward putting the item marker in column 1. When you properly align a wrapped line to the line above it, it ends up getting indented by 4 spaces, which triggers markdown thinking it's code while editing. (It renders fine, but confuses editors.)

Reply:
Ok. The main problem was the inconsistency, so I'll change it to remove the indents.